engine/client/gl_rstrobe.c

```diff
@@ -127,21 +146,61 @@ _inline qboolean _isPositive( void )
 	return __checkState( PHASE_POSITIVE );
 }
 
+static int _frameCount( CountType type )
```

Comment:
You may declare it as inline, so compiler will replace call with appropriate case-block.

Reply:
Agreed. But I don't have much time right now to create commits to resolve these things, can you do that ? Otherwise I will close this pull request and reopen later.

Reply:
Oh, okay then. :)